streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/configuration/ConfigFileType.java

```diff
@@ -22,12 +22,12 @@
  * @see ConfigFileWriter
  */
 public enum ConfigFileType {
-  HADOOP_XML, PROPERTIES, YAML;
+  HADOOP_XML, PROPERTIES, YAML, ZOOKEEPER_CFG;
 
   public static ConfigFileType getFileTypeFromFileName(String fileName) {
     String fileExt = Files.getFileExtension(fileName);
     if (fileExt.isEmpty()) {
-      return null;
+      throw new IllegalArgumentException("Not supported config file extension: " + fileExt);
```

Comment:
1. I just picked `null` as return value but not relevant with NPE. It's not happening because of referencing null.
2. I'm not in favor of throwing custom NPE since NPE doesn't have any meaning other than referencing null, and no one would want to catch NPE in normal.